### PR TITLE
LuaFAR: update luautf8 library to 0.1.5

### PR DESCRIPTION
**Pull request by johnd0e**
New functions:
   - utf8.isvalid(s) -> boolean
   - utf8.clean(s[, replacement_string]) -> cleaned_string, was_valid
   - utf8.invalidoffset(s[, init]) -> offset

   See https://github.com/starwing/luautf8 for more info.

   Note:
     utf8.utf8valid is rather obsolete now.

### CODE REVIEW

**Comment by johnd0e** [comment]
@shmuz

**Comment by shmuz** [comment]
@johnd0e , если macrotest.lua даёт OK, то я тоже.

File: enc/enc_lua/luafar_manual.tsi [hunk 1]
```diff
@@ -11720,6 +11741,8 @@ mtime=3776451338
 #_
 #_**Note:**
 #_  This is an added function, not present in the original **luautf8** library.
+#_  It was introduced in LuaFAR before utf8.isvalid/utf8.invalidoffset got available in lua-utf8, and now is rather obsolete.
+#_  Left for compatibility.
```
Comment (johnd0e) [inline enc/enc_lua/luafar_manual.tsi:11745]: @shmuz 
Is this wording appropriate?

Comment (shmuz) [reply enc/enc_lua/luafar_manual.tsi:11745]: Yes.

**Pull request merged**